tests/unit_tests/test_cli.py

```
@@ -0,0 +1,274 @@
+    def test_collection_delete_requires_confirmation(self):
+        """Test that delete requires --yes flag or confirmation"""
+        from pyseekdb.cli import main
+        runner = CliRunner()
+
+        with patch("pyseekdb.cli.Client") as mock_client_class:
+            mock_client = MagicMock()
+            mock_client_class.return_value.__enter__ = MagicMock(return_value=mock_client)
+            mock_client_class.return_value.__exit__ = MagicMock(return_value=False)
+
+            # Without --yes, should prompt (input 'n' to abort)
+            result = runner.invoke(main, ["collection", "delete", "test_coll"], input="n\n")
```

Ruff (0.14.14)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)